tox.ini

```diff
@@ -17,21 +17,18 @@ deps =
     -rtest-requirements.txt
     ansible22: ansible==2.2.1.0
 commands =
-    unit: py.test -vv -x --cov={toxinidir}/molecule/ {posargs}
+    unit: py.test -vv --cov-report=term-missing --cov={toxinidir}/molecule/ --no-cov-on-fail {posargs}
     functional: py.test -vv -x test/functional/ {posargs}
+    lint: flake8
```

Comment:
moves the lint test to the main [testenv] stanza so that it can be invoked by tox-travis